### DISCUSSION TITLE
Blurred Image saved :: Object Detection example

### DISCUSSION THREAD

**Discussion by AmarRobo** [AxisCommunications/discussions#146] [ACAP]
I am trying the object detection example [acap-native-sdk-examples].(https://github.com/AxisCommunications/acap-native-sdk-examples). The saved crop image on the device in the folder /var/volatile/ tmp/ is blurred. I tried with default quality 80.  
**![detection_0](https://user-images.githubusercontent.com/27479245/209511377-9ba84349-3c0c-4587-836e-9fcdda7f9bf9.jpg)**
I tried saving a 640 X 640 from the raw stream using jpeg buffer with image quality = 100 and the saved image (/tmp/crop.jpg) looks like

            unsigned char* crop_buffer1 = crop_interleaved(cropAddr, args.raw_width, args.raw_height, CHANNELS,
                                                                0, 0, 640, 640);
            unsigned long jpeg_size = 0;
            unsigned char* jpeg_buffer = NULL;
            struct jpeg_compress_struct jpeg_conf;
            set_jpeg_configuration(640, 640, CHANNELS, 100, &jpeg_conf);
            buffer_to_jpeg(crop_buffer1, &jpeg_conf, &jpeg_size, &jpeg_buffer);
            char file_name[32];
            snprintf(file_name, sizeof(char) * 32, "/tmp/crop.jpg");
            jpeg_to_file(file_name, jpeg_buffer, jpeg_size);
 
![crop](https://user-images.githubusercontent.com/27479245/209511892-0a5df378-83ba-42d4-af52-f6f904f892e0.jpg)
Is it normal output or is there any way to save a good image?
**Environment:**
Axis device model: P1375-E
Axis device firmware version: [Axis OS 10.08.1]
Stack trace or logs: [e.g. Axis device system logs]
OS and version: [ Ubuntu 20.04.03 LTS]

**Comment by vivekatoffice** [comment]
Related links:
https://github.com/orgs/AxisCommunications/discussions/530
